### PR TITLE
Drop the auto-JSON confusion

### PR DESCRIPTION
**Pull request by jcorbin**

### CODE REVIEW

**Comment by Raynos** [comment]
Please fix examples, benchmarks and README too.

**Comment by jcorbin** [comment]
All examples were already compliant as far as I could see, please confirm.

**Comment by Raynos** [comment]
@jcorbin confirmed. You are correct.

File: v1/frame.js [hunk 1]
```diff
@@ -54,34 +54,23 @@ TChannelFrame.prototype.set = function (arg1, arg2, arg3) {
 
     if (Buffer.isBuffer(arg2)) {
         this.arg2 = arg2;
+    } else if (typeof arg2 === 'string') {
+        this.arg2 = new Buffer(arg2);
+    } else if (arg2 === null || arg2 === undefined) {
+        this.arg2 = emptyBuffer;
```
Comment (kriskowal) [inline v1/frame.js]: emptyBuffer is not defined.

**Pull request merged**